src/surfaceComponent/Card/Card.tsx

```diff
@@ -0,0 +1,37 @@
+import { PropsWithChildren } from "react";
+import * as React from "react";
+import imageUrl from "../../assets/card/card.png";
+import * as styles from "./Card.module.css";
+import classNames from "classnames/bind";
+
+const cx = classNames.bind(styles);
+
+
+interface CardProps {
+  fullWidth?: boolean;
+}
+
+export const CardImage = () => {
+  return (
+    <div className={styles.imageWrapper}>
+      <img src={imageUrl} alt="placeholder" />
```

Comment:
**nitpick:** Yea, Nerdy is right here - we should be able to parametrize this `alt`, as it is very bad solution for SEO when we have non descriptive image alts. However, this component looks un parametrized at all. We can't set image `url` which makes this component useless. We need to fix it